pythonFeed.sh

```diff
@@ -7,6 +7,7 @@ else
   exit 1
 fi
 
+vers=()
```

Comment:
where is this being used?

Reply:
This was a mistake that I forgot to remove thanks for catching it.